sqlite/sqlite.go

```diff
@@ -838,6 +839,7 @@ func (db *DB) NewVoucher(ctx context.Context, ov *fdo.Voucher) error {
 		"guid":        ov.Header.Val.GUID[:],
 		"device_info": ov.Header.Val.DeviceInfo,
 		"cbor":        data,
+		"status":      int(fdo.VoucherStatusReady),
```

Comment:
The table `mfg_vouchers` does not have this column, so `NewVoucher` will fail without auto-extend in `BeforeVoucherPersist`.